cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1,24 +1,268 @@
 use std::collections::{BTreeMap, HashMap};
+use std::ops::{Deref, DerefMut};
 use std::process::Command;
 use std::str::FromStr;
 
+use indoc::{formatdoc, indoc};
 use log::debug;
 use serde::de::Error;
 use serde::{Deserialize, Serialize};
-use toml_edit::{self, DocumentMut, Formatted, InlineTable, Item, Table, Value};
-
+use toml_edit::{
+    self,
+    value,
+    Array,
+    Decor,
+    DocumentMut,
+    Formatted,
+    InlineTable,
+    Item,
+    Key,
+    Table,
+    Value,
+};
+
+use super::environment::path_environment::InitCustomization;
 use crate::data::{System, Version};
 use crate::models::pkgdb::PKGDB_BIN;
 
 pub(super) const DEFAULT_GROUP_NAME: &str = "toplevel";
 pub(super) const DEFAULT_PRIORITY: usize = 5;
 
+/// Represents the `[install]` table key in manifest.toml
+pub const MANIFEST_INSTALL_TABLE_KEY: &str = "install";
```

Comment:
Nitpick: I don't think you need the type in the name of the constant

Reply:
I didn't know this before your comment but Rust requires constants to be explicitly typed: https://doc.rust-lang.org/rust-by-example/custom_types/constants.html

Reply:
Correct

Reply:
Having re-read your first comment I think you were referring to the "TABLE" part inside the const identifier. Is this correct?